packaging/ondemand.spec

```diff
@@ -112,7 +113,12 @@ EOS
 scl enable ondemand - << \EOS
 set -x
 set -e
-rake install PREFIX=%{buildroot}/opt/ood
+rake --trace install PREFIX=%{buildroot}/opt/ood OBJDIR=$(pwd)/build
```

Comment:
here too I'm surprised that this is equivalent to `PREFIX=%{buildroot}/opt/ood OBJDIR=$(pwd)/build rake --trace install`

Reply:
We've been setting `PREFIX` after for ages, works either way.